dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1322,6 +1353,27 @@ void TeachingTip::RevokeViewportChangedEvent()
     m_targetLayoutUpdatedRevoker.revoke();
 }
 
+void TeachingTip::WindowSizeChanged(const winrt::CoreWindow&, const winrt::WindowSizeChangedEventArgs&)
+{
+    RepositionPopup();
+}
+
+void TeachingTip::XamlRootChanged(const winrt::XamlRoot&, const winrt::XamlRootChangedEventArgs&)
+{
+    if (winrt::IUIElement10 uiElement10 = *this)
+    {
```

Comment:
Nitpick: shouldn't need to check for IUIElement10 here, since this handler can only be called when IUIElement10 is implemented.